src/components/graph/Graph.tsx

```diff
@@ -132,11 +132,11 @@ export const Graph: FC<IGraphProps> = (props: IGraphProps) => {
       nodeMap.getSimulationNodeDatums().forEach(node => {
         if (node.id.indexOf(CONST.LINK_NODE_PREFIX) !== -1) {
           node.x =
-            (nodeMap.get(node.id.split("-")[1]).force.x ?? 0) * 0.5 +
-            (nodeMap.get(node.id.split("-")[2]).force.x ?? 0) * 0.5;
+            (nodeMap.get(node.id.split("&")[1]).force.x ?? 0) * 0.5 +
```

Comment:
Can you extract it as a const, so that in future, if we want to change the delimiter symbol again, we will not need to update at every place.

Reply:
Sure, so using "delimiter" as new delimiter, right?

Reply:
> Can you extract it as a const, so that in future, if we want to change the delimiter symbol again, we will not need to update at every place.

Sure, so using "delimiter" as new delimiter, right?